src/main/java/com/revlearn/team1/model/DiscussionBoard.java

```diff
@@ -0,0 +1,25 @@
+package com.revlearn.team1.model;
+
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class DiscussionBoard {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long discussionBoardId;
+
+    @ManyToOne
+    @JoinColumn(name="course_id")
+    private Course course;
```

Comment:
see change requests in the other PR you submitted. Once that one gets merged in, this shouldn't be a problem tho